docs/js/features/odata/odata-v2-client.mdx

```diff
@@ -338,6 +338,27 @@ BusinessPartner.requestBuilder()
   .execute(destination);
 ```
 
+## Function imports
+### Request builder
+The [function imports request builder](pathname:///api/1.28.1/classes/sap_cloud_sdk_core.functionimportrequestbuilderv2) helps build a request for a service operation with essential parameters.
+The example below creates a function import request builder for the service operation "PostGoodsIssue" and then execute it against your service.
+```ts
+postGoodsIssue({outboundDeliveryOrder: "order"})
+  .execute(destination);
+```
+The service operation is defined in the service metadata. Therefore, for some services, there might be no function imports at all like the beloved `Business Partner Service`.
```

Comment:
I would swapt the order. Fist say where the `PostGoddIssue` is loacated (the warehou servie). And them make something like:

`Note that not all OData service contain function imports. For example the BP service does not. bla bla bla`. I also would remove the beloved.